FND/Relations/Relations.rdf

```diff
@@ -81,8 +81,7 @@ the FIBO FND RFC, was revised in advance of the September 2013 New Brunswick, NJ
 	<owl:ObjectProperty rdf:about="&fibo-fnd-rel-rel;appliesTo">
 		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-rel-rel;refersTo"/>
 		<rdfs:label>applies to</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-fnd-rel-rel;Reference"/>
-		<skos:definition>a relation indicating something that is pertinent or relevant to the concept</skos:definition>
+		<skos:definition>is pertinent, suitable, or relevant to</skos:definition>
```

Comment:
too broad compared to its actual usage in FIBO

Reply:
There are actually 77 uses of the property appliesTo in FIBO.  Some of those uses can be replaced with a more specific property, I agree, but I don't know if there is a replacement that is more specific available for every one of them.  I would rather not make a change to this until we have (1) replaced some of its uses with appropriately more specific properties, and (2) done further research on the remainder to decide whether there is a better definition that could reflect its usage for all of them.  The main change here was to eliminate "a relation indicating something that", which is overly verbose and doesn't meet our ISO 704 compliant approach to definitions.  I didn't change the meaning at all.

Reply:
My problem here is less with the property itself, but that the proposed definition does not reflect how it's  used, and should be used.